src/utils/crypto.ts

```diff
@@ -212,8 +225,8 @@ export function verifyMessage (str, signature, publicKey) {
  * @param {Buffer} publicKey - Public key to verify
  * @return {Boolean} Valid?
  */
-export function isValidKeypair (privateKey, publicKey) {
+export function isValidKeypair (privateKey: Buffer, publicKey: Buffer): boolean {
   const message = Buffer.from('TheMessage')
   const signature = sign(message, privateKey)
-  return verify(message, signature, publicKey)
+  return verify(message, <Buffer>signature, publicKey)
```

Comment:
```suggestion
  return verify(message, signature, publicKey)
```

aligns with previous comment of changing the type of the second parameter of the `verify` function